flink-connectors/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseWriteStreamExample.java

```diff
@@ -87,22 +90,22 @@ public void configure(Configuration parameters) {
 
 		@Override
 		public void open(int taskNumber, int numTasks) throws IOException {
-			table = new HTable(conf, "flinkExample");
+			Connection connection = ConnectionFactory.createConnection(conf);
```

Comment:
I think we should add try / catch block to protect the connect leak.

Reply:
Based on [HBase Connection JavaDoc](https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/Connection.html#close--) it seems the caller should invoke `close` method to release resource? so I suggest we should close connection in udf's `close` method.